amida-tech/amida-auth-microservice
# [WIP] password reset

#### What's this PR do?
Adds password reset functionality, including sending emails with a reset link.

#### Related JIRA tickets:
SER-8

#### How should this be manually tested?
For now, by running the test harness. You can submit API requests to go through the workflow, but make sure to set `NODE_ENV=test`.

#### Any background context you want to provide?
Currently, the test to actually update the password with the reset token is failing. It looks like two things are going on. (1) in the unit test, running `user.reload()` doesn't pull in the password change. (2) it looks like the password is getting changed, but the new `salt` is not getting saved, and I am not sure why.
